File: Web/Divyanshu-Vareshney_divyanshuvarshney_2125it1075_2/GettingStarted/Count Digits In A Number.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
    
    //write your code here
    int d=0;
    while(n>0){
        n=n/10;
        d++;
    }
    cout<<d;
    //DONE
    
}

File: Android/aryanmishra29_AryanMishra_2125csme1029_2/GettingStarted/Count_Digits_in_a_Number.java
import java.util.Scanner;

public class Count_Digits_in_a_Number{
    public static void main(String[] args) {
        // write your code here
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int d=0;
        while(n>0){
            d++;
            n/=10;
        }
        System.out.println(d);
       }
}